CONTRIBUTING.md

```diff
@@ -59,35 +59,55 @@ def count_truthy(elements: List[Any]) -> int:
-
-Kedro-Viz uses an environment variable to configure the data source. You can set it when starting up the dev server, e.g. `DATA=random npm start` will serve random procedurally-generated data (refreshed on each page-load). This is usually the most useful setting for local development.
+```bash
+npm install
+```
 
-In other words, to run the app in development mode on a local server, use one of the following:
+Now you're ready to begin development. Start the development server:
 
-- `DATA=random npm start` --> Serve randomly-generated data
-- `DATA=animals npm start` --> Serve example test data, from `/src/utils/data.mock.js`
-- `DATA=lorem npm start` --> Serve example test data, from `/src/utils/data.mock.js`
-- `npm start` --> Serve data loaded from `/public/logs/nodes.json`
+```bash
+npm start
+```
 
-This will serve the app at [localhost:4141](http://localhost:4141/), and watch files in `/src` for changes. It will also update the `/lib` directory, which contains a Babel-compiled copy of the source. This directory is exported to `npm`, and is used when importing as a React component into another application. It is updated automatically on save in case you need to test/debug it locally (e.g. with `npm link`). You can also update it manually, by running
+This will serve the app at [localhost:4141](http://localhost:4141/), and watch files in `/src` for changes. It will also update the `/lib` directory, which contains a Babel-compiled copy of the source. This directory is exported to `npm`, and is used when importing as a React component into another application. It is updated automatically when you save in case you need to test/debug it locally (e.g. with `npm link`). You can also update it manually, by running
 
 ```bash
 npm run lib
 ```
 
+#### Data sources
+
+Kedro-Viz uses a unique identifier to determine the data source. You can configure this by appending a query string to the URL, e.g. `http://localhost:4141/?data=random`. Alternatively, you can set it with an environment variable when starting up the dev server:
+
+```bash
+DATA=random npm start
+```
+
+There are several different data sources available. By default in production, the app asynchronously loads JSON from the `/api/nodes.json` endpoint, and you can replicate this in development by placing a dataset at `/public/api/nodes.json`. Alternatively, you can use one of the mock unit-testing/demo datasets, or pseudo-random data procedurally-generated on page load, which is often useful for local development.
+
+| Identifier | Data source |
+|------------|-------------|
+| `json` (default) | `/public/api/nodes.json` |
+| `random` | Randomly-generated data |
+| `demo` | `/src/utils/data/demo.mock.js` |
+| `lorem` | `/src/utils/data/lorem-ipsum.mock.js` |
+| `animals` | `/src/utils/data/animals.mock.js` |
+| `layers` | `/src/utils/data/layers.mock.js` |
```

Comment:
```suggestion
| Identifier       | Data source                           |
| ---------------- | ------------------------------------- |
| `json` (default) | `/public/api/nodes.json`              |
| `random`         | Randomly-generated data               |
| `demo`           | `/src/utils/data/demo.mock.js`        |
| `lorem`          | `/src/utils/data/lorem-ipsum.mock.js` |
| `animals`        | `/src/utils/data/animals.mock.js`     |
| `layers`         | `/src/utils/data/layers.mock.js`      |
```

Reply:
This was part of #181 - this is definitely not the appropriate PR to be making these sorts of changes. This is just a release approval PR, we shouldn't be using it to make code/documentation changes other than release notes.